<launch>
    <arg name="wheelchair_max_rotation_speed"       default="90.0" />  <!-- 90 degree -->
    <arg name="wheelchair_rotation_speed_inctl"     default="5.0"  />  <!-- increase 5 degree per keypress -->
    <arg name="wheelchair_max_forward_speed"        default="2.0"  />  <!--  2 m/s    -->
    <arg name="wheelchair_forward_speed_inctl"      default="0.1"  />  <!-- incerase 0.1 m/s per keypress -->

    <node pkg="wheelchair_keyboard_control" type="wheelchair_keyboard_control.py" name="wheelchair_keyboard_control" output="screen">
        <param name="wheelchair_max_rotation_speed"         type="double"  value="$(arg wheelchair_max_rotation_speed)"/>
        <param name="wheelchair_rotation_speed_inctl"       type="double"  value="$(arg wheelchair_rotation_speed_inctl)"/>
        <param name="wheelchair_max_forward_speed"          type="double"  value="$(arg wheelchair_max_forward_speed)"/>
        <param name="wheelchair_forward_speed_inctl"        type="double"  value="$(arg wheelchair_forward_speed_inctl)"/>
    </node>
</launch>